Apps/phadldap/adldap.json

```diff
@@ -0,0 +1,1311 @@
+                    "data_type": "string",
+                    "example_values": [
+                        "ObjectGUID"
+                    ]
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "Total objects: 2"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.entries.*.attributes",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.entries.*.dn",
+                    "data_type": "string",
+                    "example_values": [
+                        "CN=SVC-SPLUNK,OU=splunk,DC=SPLUNK,DC=LAB"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.entries.*.attributes.objectGUID",
+                    "data_type": "string",
+                    "example_values": [
+                        "{a6c536dd-2487-41dd-8524-0037342505da}"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        2
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "custom",
+                "width": 10,
+                "height": 5,
+                "view": "query_view.display_attributes",
+                "title": "AD Query"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "set attribute",
+            "identifier": "set_attribute",
+            "description": "Add, delete, or replace an attribute of a user",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {
+                "use_samaccountname": {
+                    "description": "Use sAMAccountName instead of distinguishedName",
```

Comment:
Please be consistent with uppercase/lowercase in `sAMAccountName` throughout the app